pkg/entities/webhook.go

```diff
@@ -139,6 +139,22 @@ func (e *Entity) HandleDiscordMessage(message *discordgo.Message) (*response.Han
 		channelID = message.ChannelID
 	)
 
+	if _, err := e.repo.Users.GetOne(discordID); err == gorm.ErrRecordNotFound && message.Content != "" {
```

Comment:
`message.Content != ""`
what is this check for?

Reply:
`message.Content = ""` is default message when someone join server, event `guildMemberAdd` was handle for create user.

Reply:
but this func would only be called for event `messageCreate` right?

Reply:
default message will call event `messageCreate` to and do nothing. I have new update for this